### ISSUE TITLE
tqdm.pandas() not working under pandas 0.25.0

### ISSUE THREAD

**Issue by martinfleis** [tqdm/tqdm#780]
Under new pandas 0.25.0, `tqdm.pandas()` will cause ImportError probably due to the changes of `Groupby`behaviour in pandas (https://pandas.pydata.org/pandas-docs/stable/whatsnew/v0.25.0.html). Full error message below. Issue is the same under Linux as well as macOS, tqdm version is  '4.32.2'. Detail are here https://travis-ci.org/martinfleis/momepy/jobs/561121151/.

    Traceback (most recent call last):
      File "/Users/martin/anaconda3/envs/momepy37/lib/python3.7/site-packages/tqdm/_tqdm.py", line 613, in pandas
        from pandas.core.groupby.groupby import DataFrameGroupBy, \
    ImportError: cannot import name 'DataFrameGroupBy' from 'pandas.core.groupby.groupby' (/Users/martin/anaconda3/envs/momepy37/lib/python3.7/site-packages/pandas/core/groupby/groupby.py)

    During handling of the above exception, another exception occurred:

    Traceback (most recent call last):
      File "<stdin>", line 1, in <module>
      File "/Users/martin/anaconda3/envs/momepy37/lib/python3.7/site-packages/tqdm/_tqdm.py", line 616, in pandas
        from pandas.core.groupby import DataFrameGroupBy, \
    ImportError: cannot import name 'PanelGroupBy' from 'pandas.core.groupby' (/Users/martin/anaconda3/envs/momepy37/lib/python3.7/site-packages/pandas/core/groupby/__init__.py)

**Comment by will-holley** [comment]
+1

**Comment by casperdcl** [comment]
yes looks like API changed with `pandas==0.25.0` https://travis-ci.org/tqdm/tqdm/jobs/560783546

**Comment by whyboris** [comment]
Confirming:

 - Python `3.7.4`
 - tqdm `4.32.2`
 - pandas `0.25.0`

```py
import pandas as pd
from tqdm import tqdm
tqdm.pandas()
```
The above code is enough to cause this:
```
/Users/.../python3.7/site-packages/tqdm/_tqdm.py:603: FutureWarning: The Panel class is removed from pandas. Accessing it from the top-level namespace will also be removed in the next 
version
  from pandas import Panel
Traceback (most recent call last):
  File "/Users/.../python3.7/site-packages/tqdm/_tqdm.py", line 611, in pandas
    from pandas.core.groupby.groupby import DataFrameGroupBy, \
ImportError: cannot import name 'DataFrameGroupBy' from 'pandas.core.groupby.groupby' (/Users/.../python3.7/site-packages/pandas/core/groupby/groupby.py)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "pyetl.py", line 32, in <module>
    tqdm.pandas()
  File "/Users/.../python3.7/site-packages/tqdm/_tqdm.py", line 614, in pandas
    from pandas.core.groupby import DataFrameGroupBy, \
ImportError: cannot import name 'PanelGroupBy' from 'pandas.core.groupby' (/Users/.../python3.7/site-packages/pandas/core/groupby/__init__.py)
```
(paths above shortened with `...` for brevity)

**Comment by wlockiv** [comment]
I am experiencing the issue as described as well.

**Comment by casperdcl** [comment]
yes thank you `pandas` for breaking API while still having a major version number of `0`. Beta mode, right? Should be fixed soon.

**Comment by casperdcl** [comment]
fixed in `tqdm==v4.33.0`

**Comment by ll-DS** [comment]
+1
python = 3.6.8
pandas = 0.25.1
tqdm = 4.35.0
